modules/md5sum/main.nf

```diff
@@ -0,0 +1,35 @@
+process MD5SUM {
+    tag "$meta.id"
+    label 'process_low'
+
+    if (params.enable_conda) {
+        exit 1, "Conda environments cannot be used when using bcl-convert. Please use docker or singularity containers."
+    }
+    container "debian:bullseye-slim"
```

Comment:
not so sure about the container definition. What about conda? harshil has been using this multi-purpose container: https://github.com/nf-core/modules/blob/master/modules/untar/main.nf maybe worth to try to keep it consistent